README.adoc

```diff
@@ -23,57 +23,26 @@ container which is build afterwards. This script also makes sure all required su
 This section describes how to execute the Cyface data collector.
 
 === Running from Docker
-After you did build the system (see Building above) simply run `start.sh` inside the root of this project. This calls docker to bring up two Mongo database containers and a container running the Cyface data collector API.
+After you did build the system (see Building above) simply run `docker-compose up` inside the root of this project. This calls docker to bring up two Mongo database containers and a container running the Cyface data collector API.
 In addition it starts https://prometheus.io/[Prometheus] and https://grafana.com/[Grafana] as monitoring tools for the running server.
 
 Prometheus stores all the metrics data, while Grafana provides a dashboard for visualization.
 Per default the Grafana dashboard is available via port 3000. The Collector API is by default available via port 8080.
 
 This means if you boot up everything using the default settings you can access Grafana via `http://localhost:3000`.
 The default username is *cyface-admin* and the default password *cyface-admin@grafana*.
-The Collector API is accessable via `https://localhost:8080/api/v2`.
+The Collector API is accessable via `http://localhost:8080/api/v2`.
 
 ==== Certificates
 The Cyface Data Collector provides keys you may use for testing and during development. 
```

Comment:
Ich verstehe das so: Man kann durch ./build.sh und docker-compose up den Dienst bereits nutzen (unsichere keys). Das klappt bei mir auf dem S1 leider nicht (ich wollte die test V2 API die ich bereits aufgesetzt hatte durch die neuere Version hier ersetzen um die Änderungen zu testen):

`collector_api | Dec 18, 2018 9:40:19 AM de.cyface.collector.Application
collector_api | INFO: Enabling metrics capturing to prometheus!
collector_api | Dec 18, 2018 9:40:20 AM de.cyface.collector.Parameter
collector_api | INFO: Read json value {"db_name":"cyface-data","connection_string":"mongodb://mongo-data:27017","data_source_name":"cyface-data"} for key mongo.datadb
collector_api | Dec 18, 2018 9:40:20 AM de.cyface.collector.Parameter
collector_api | INFO: Using configuration value: /app/secrets/jwt/public.pem for key: jwt.public.
collector_api | java.io.FileNotFoundException: /app/secrets/jwt/public.pem (No such file or directory)
collector_api |         at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
collector_api |         at java.lang.Thread.run(Thread.java:748)
collector_api | Dec 18, 2018 9:40:20 AM io.vertx.core.impl.launcher.commands.VertxIsolatedDeployer
collector_api | SEVERE: Failed in deploying verticle
collector_api | java.io.FileNotFoundException: /app/secrets/jwt/public.pem (No such file or directory)
collector_api |         at java.io.FileInputStream.open0(Native Method)
collector_api |         at java.io.FileInputStream.open(FileInputStream.java:195)
collector_api |         at java.io.FileInputStream.<init>(FileInputStream.java:138)
collector_api |         at java.io.FileInputStream.<init>(FileInputStream.java:93)
collector_api |         at de.cyface.collector.MainVerticle.extractKey(MainVerticle.java:232)
collector_api |         at de.cyface.collector.MainVerticle.start(MainVerticle.java:80)
`

Reply:
Ich habe nun mal die keys wie unten unter dem Link beschrieben generiert und in secrets gelegt:
`16393961 -rw-------  1 armin armin  323 Dez 18 10:54 private_key.pem
16393963 -rw-r--r--  1 armin armin  138 Dez 18 10:54 public.pem
`

Und bekomme weiterhin den Fehler:

```
collector_api | Dec 18, 2018 9:55:59 AM de.cyface.collector.Parameter
collector_api | INFO: Read json value {"db_name":"cyface-data","connection_string":"mongodb://mongo-data:27017","data_source_name":"cyface-data"} for key mongo.datadb
collector_api | Dec 18, 2018 9:55:59 AM de.cyface.collector.Parameter
collector_api | INFO: Using configuration value: /app/secrets/jwt/public.pem for key: jwt.public.
collector_api | java.io.FileNotFoundException: /app/secrets/jwt/public.pem (No such file or directory)
collector_api |         at java.io.FileInputStream.open0(Native Method)
collector_api |         at java.io.FileInputStream.open(FileInputStream.java:195)
collector_api |         at java.io.FileInputStream.<init>(FileInputStream.java:138)
collector_api |         at java.io.FileInputStream.<init>(FileInputStream.java:93)
collector_api |         at de.cyface.collector.MainVerticle.extractKey(MainVerticle.java:232)
collector_api |         at de.cyface.collector.MainVerticle.start(MainVerticle.java:80)
collector_api |         at io.vertx.core.impl.DeploymentManager.lambda$doDeploy$8(DeploymentManager.java:494)
collector_api |         at io.vertx.core.impl.ContextImpl.executeTask(ContextImpl.java:320)
collector_api |         at io.vertx.core.impl.EventLoopContext.lambda$executeAsync$0(EventLoopContext.java:38)
collector_api |         at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163)
collector_api |         at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:404)
collector_api |         at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:462)
collector_api |         at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:897)
collector_api |         at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
collector_api |         at java.lang.Thread.run(Thread.java:748)
collector_api | Dec 18, 2018 9:55:59 AM io.vertx.core.impl.launcher.commands.VertxIsolatedDeployer
collector_api | SEVERE: Failed in deploying verticle
```

Folgendes habe ich (noch) nicht gemacht da nicht klar, ob das auch hier zutrifft, sollte aber ja nicht die Ursache sein, wieso er die Dateien nicht findet.

> Do note that all the lines -----BEGIN …​ and -----END…​ should be stripped from the string to be passed to the configuration.